spec/02-integration/20-wasm/09-filter-meta_spec.lua

```diff
@@ -457,8 +457,6 @@ describe("filter metadata [#off] yaml config", function()
                 append:
                   headers:
                   - x-response-transformer-2:TEST
-                rename: ~
-                remove: null
```

Comment:
Could you elaborate on this. Is there something the explicit `null` breaks? Or just breaks the tests? The explicit nulls are normally removed as `process_auto_fields` removes them by default on read.

The explicit `null` basically means that `read` should not `fill` default, while `missing` (nil), means that default can be filled on read. Do we have a tests that confirms this? If not, could we add one (edit: oh, it looks like you added one in this pr)? In this case the `rename` and `remove` does not have defaults, right, so it should not matter?

I read this:
https://github.com/Kong/kong-ee/pull/7680#issuecomment-1882260870

But I cannot fully understand it (as explained above). Is this a workaround to some other issue somewhere else? @flrgh?

I struggle to understand what `response_transformer_with_schema` is and where its schema is specified (perhaps it has `defaults`?).

Reply:
The two lines break the tests. Maybe due to the diff between WasmX and OpenResty Gateway.

Have little info on WasmX part. A reference link: https://github.com/Kong/kong-ee/pull/7680#issuecomment-1882260870

Reply:
@outsinre, I added comment to slack thread, and pointed it to @flrgh to answer if further steps are needed with filter JSON parsing to properly support `null`. In  a meanwhile I am marking this as resolved and will approve the pr.